run.go

```diff
@@ -46,9 +47,21 @@ func main() {
 	flag.StringVar(&dataDir, "data", "./data", "directory where cerca will dump its database")
 	flag.Parse()
 	if len(sessionKey) == 0 {
-		complain("please pass a random session auth key with --authkey")
-	} else if len(allowlistLocation) == 0 {
-		complain("please pass a file containing the verification code domain allowlist")
+		if !dev {
+			complain("please pass a random session auth key with --authkey")
+		}
+		sessionKey = "0"
+	}
+	if len(allowlistLocation) == 0 {
+		if !dev {
+			complain("please pass a file containing the verification code domain allowlist")
+		}
+		allowlistLocation = "temp.txt"
+		if _, err := os.Stat(allowlistLocation); errors.Is(err, os.ErrNotExist) {
+			if _, err := os.Create(allowlistLocation); err != nil {
+				complain(fmt.Sprintf("couldn't create ./temp.txt: %s", err))
+			}
+		}
 	}
```

Comment:
can you use [`util.CreateIfNotExist`](https://github.com/cblgh/cerca/blob/main/util/util.go#L171-L191) & use the error it returns there to `complain`? 

also since we're now creating a file automatically, how about using the name `temp-allowlist.txt` instead of `temp.txt`? feels like it would be less surprising at least! feel half tempted to also suggesting tossing in a `fmt.Println` mentioning the creation of that file for that same reason